src/rules/mixins-no-risky-parent-selectors/README.md

```diff
@@ -0,0 +1,133 @@
+# mixins-no-risky-parent-selectors
+
+
+Parent selectors are not reliable in the following contexts:
+
+- Within a @mixin rule.
+- Nested within another style rule.
+- When they are not positioned at the beginning of a complex selector.
+- Within a @at-root rule.
```

Comment:
In the issue I filed, I said that it _shouldn't_ be considered an error within `@at-root`. However, I think I was wrong about that—`&` within `@at-root` is still affected by outer parent selectors, so I think there should be no special handling for `@at-root` in either direction.